research-software-directory/RSD-as-a-service
# 622: show only specific links in the organisation info card

# Show only specific links in the organisation card

Closes #622

Changes proposed in this pull request:
* The links in the organisation card are selectively applied because the links received from ROR api are in some cases outdated (use http instead of https). We only show: first website link, link to ROR info and, if provided, wikipedia link.
* Instead of showing links we show an icon and the label: Website, ROR info and Wikipedia
* New logic selects the links in the following way: 
    * website link: if the website property is present in the organisation in RSD we use that link, otherwise we use first entry from the links array returned by ROR api. If both are not available we do not show website link entry.
   * ROR info: if ror_id is provided by ROR api we show this link as "ROR info" (id is always present)
   * Wikipedia: if wikipedia link is provided by ROR api we show this entry under "Wikipedia"
* Organisation type label is made (slightly) smaller by reducing the font size
* The color of organisation type label remains unchanged (it is primary color). It can be made neutral (which has less flexibility when changing the theme colors). See second printscreen and let me know what you prefer.
* **Unrelated change**: commit graph message changed to "We cannot display the activity graph, because we do not know the repository url."

How to test:
* `make start` to restart everything, when you done use `docker-compose down`
* login as rsd_admin to be able to edit existing project/software, otherwise create new software/project
* add organisation, for example eScience Center (with logo), VU or other
* verify the organisation card entries
* any suggestions are welcome!

## Example VU organisation card
![image](https://user-images.githubusercontent.com/9204081/199300501-604582cd-a901-4df6-9dfe-8326b95a6d16.png)

## Example organisation type with neutral color
![image](https://user-images.githubusercontent.com/9204081/199299732-6c29598a-8b8c-45f5-9240-a7689c29a66a.png)

PR Checklist:
